addons/calendar/models/calendar_event.py

```diff
@@ -322,10 +322,10 @@ def _compute_stop(self):
         # its recomputation. To avoid this we manually mark the field as computed.
         duration_field = self._fields['duration']
         self.env.remove_to_compute(duration_field, self)
-        for event in self.filtered('duration'):
+        for event in self:
             # Round the duration (in hours) to the minute to avoid weird situations where the event
             # stops at 4:19:59, later displayed as 4:19.
-            event.stop = event.start + timedelta(minutes=round((event.duration or 1.0) * 60))
+            event.stop = event.start + timedelta(minutes=round((event.duration) * 60)) if event.duration else event.start
```

Comment:
if no duration, stop date will be exactly the same than start date, before it was at least 1 hours later. is it intended ?

Reply:
@dbeguin  previously, if there's no duration, it was never possible to compute stop date at least one hour later than start date because of the [filtration](https://github.com/odoo/odoo/pull/54673/files/87fcd76f8f8a5b9384eaf29049d3d8976e7960cd#diff-9cce8addc83a442797bce406f4dadc83L325) which we removed here. And the filtration has to be removed to fix this one, as explained in the commit message.

Reply:
ok but why not do
`event.start + timedelta(minutes=round((event.duration or 1) * 60))` as it was done before ?
if no duration, we fallback to 1h duration.